testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py

```diff
@@ -96,20 +99,21 @@ def test_verify_invalid_maskPassword_in_logs(params_from_base_test_setup, invali
     db = params_from_base_test_setup["db"]
     cbl_db = params_from_base_test_setup["source_db"]
     sync_gateway_version = params_from_base_test_setup["sync_gateway_version"]
-    log_file = params_from_base_test_setup["log_filename"]
+    log_file = params_from_base_test_setup["test_db_log_file"]
     liteserv_platform = params_from_base_test_setup["liteserv_platform"]
+    testserver = params_from_base_test_setup["testserver"]
+    log_filename = params_from_base_test_setup["log_filename"]
 
-    num_cbl_docs = 5
+    num_cbl_docs = 50
     # Cannot run on iOS as there is no support in xcode to grab cbl logs
-    if sync_gateway_version < "2.0.0" or liteserv_platform.lower() == "ios":
-        pytest.skip('This test cannnot run with sg version below 2.0 and if platform is iOS')
+    if sync_gateway_version < "2.0.0" and log_file is not None:
```

Comment:
same as above, this condition should be only for iOS.

Reply:
same as above.